alws/crud/build_node.py

```diff
@@ -896,8 +896,9 @@ async def __update_built_srpm_url(
     # if SRPM built we need to download them
     # from pulp repos in next tasks
     if srpm_artifact and build_task.built_srpm_url is None:
+        platform_name = (await build_task.awaitable_attrs.platform).name
```

Comment:
Why do we need it?
Also, I think that's a bad idea to access attributes via `awaitable_attrs`(see https://docs.sqlalchemy.org/en/20/orm/extensions/asyncio.html#sqlalchemy.ext.asyncio.AsyncAttrs and https://docs.sqlalchemy.org/en/20/orm/extensions/asyncio.html#sqlalchemy.ext.asyncio.AsyncSession.run_sync)

Reply:
Now I'm not really sure why it was there too :/ 
BTW, you think that accessing values via `awaitable_attrs` is a bad practice or it wouldn't be right in this specific context?

Reply:
That's not a bad practice, but we should avoid such things in async apps